torch/onnx/symbolic_opset9.py

```diff
@@ -6621,6 +6636,21 @@ def prim_constant(g: jit_utils.GraphContext, *inputs, **attrs):
     )
 
 
+@_onnx_symbolic("prim::type")
+@_beartype.beartype
+def prim_type(g: jit_utils.GraphContext, device_value: _C.Value, *args, **kwargs):
+    if device_value.node().kind() == "prim::device":
+        device = jit_utils.get_device_from_value(device_value.node().input())
+        if device is not None:
+            return g.op("Constant", value_s=str(device))
```

Comment:
Thoughts on creating a specially formatted string for device types? (Maybe over complicating)